.github/workflows/check_install_script.yaml

```diff
@@ -73,7 +73,10 @@ jobs:
           pathsToTest_negative=(~/.tipi/clang/*/bin/clang
              /usr/local/share/.tipi/emsdk/*/emsdk
              /usr/local/share/.tipi/go/*/bin/go
-             /usr/local/share/.tipi/jdk/*/bin/jar)
+             /usr/local/share/.tipi/jdk/*/bin/jar
+             /usr/local/lib/cmake-re.cpp.o
+             /usr/local/lib/libz.a
+             /usr/local/lib/*.a)
```

Comment:
Thank you for adding the test on macOS, even though it is not currently relevant. This will prevent regression in the future.

Reply:
Oh yes you are right, our release package doesn't have it. 😄 Did we package the tools though ?

Reply:
On macOS, we only include the bin folder (i.e., tipi and cmake-re).